example.env

```diff
@@ -14,4 +14,5 @@ CENTRIFUGO_KEY = find-your-key
 CENTRIFUGO_ENDPOINT = https://realtime.zuri.chat/api
 # Agora APP ID and APP CERTIFICATE
 APP_ID=f910a1fb4cfe4c5996c979c54e570ba7
-APP_CERTIFICATE=04c4146b729d4bddaf9ce4ae107c9ff0
```

Comment:
Revert this please

Reply:
The value did not change, must have been a whitespace/newline my ide deleted.
The value is still untouched on line 17 below.